app/controllers/maps_controller.rb

```diff
@@ -1,6 +1,6 @@
 class MapsController < ApplicationController
   before_action :require_user, only: [:create, :update, :access, :star, :unstar, :screenshot, :events, :destroy]
-  after_action :verify_authorized, except: [:activemaps, :featuredmaps, :mymaps, :sharedmaps, :starredmaps, :usermaps]
+  after_action :verify_authorized, except: [:activemaps, :featuredmaps, :mymaps, :sharedmaps, :starredmaps, :usermaps, :livemaps, :events]
   after_action :verify_policy_scoped, only: [:activemaps, :featuredmaps, :mymaps, :sharedmaps, :starredmaps, :usermaps]
```

Comment:
Another goal of mine is to have verify_authorized happen on every controller method in the whole app. So all of these exceptions would eventually disappear, leaving just `after_action :verify_authorized`. That mostly means making policies with a bunch of functions that return true